fe/fe-core/src/test/java/org/apache/doris/nereids/util/PlanChecker.java

```diff
@@ -121,6 +122,7 @@ public PlanChecker parse(String sql) {
     public PlanChecker analyze() {
         this.cascadesContext.newAnalyzer().analyze();
         this.cascadesContext.toMemo();
+        InitMaterializationContextHook.INSTANCE.initMaterializationContext(this.cascadesContext);
```

Comment:
maybe this should judge if the `enable_materialized_view_rewrite` is true or not, and then set the  `enable_materialized_view_rewrite` in the test respectively